### PR TITLE
implement develop derivation

### PR DESCRIPTION
**Pull request by AndreyMlashkin**

### CODE REVIEW

**Review by vo-nil — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -95,6 +95,12 @@
           debug-tools = (pkgs.callPackage ./debug-tools.nix {
           });
 
+          develop = (pkgs.callPackage ./develop.nix {
```
Comment (akokoshn) [inline flake.nix:98]: Why we can't just add  flag for enable`BUILD_CRYPTO3_TESTS` and `BUILD_PARALLEL_CRYPTO3_TESTS` to proof-producer derivation? I mean add `proof-producer-all-crypto3-test` instead adding new file

**Review by akokoshn — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (AndreyMlashkin) [reply flake.nix]: done

File: flake.nix [hunk 2]
```diff
@@ -95,6 +95,12 @@
           debug-tools = (pkgs.callPackage ./debug-tools.nix {
           });
 
+          develop = (pkgs.callPackage ./proof-producer.nix {
+            enableDebug = true;
+            runTests = true;
+            sanitize = true;
```
[the remark this answers is not in the document]
Comment (AndreyMlashkin) [reply flake.nix:101]: Yes, I think it's good, you can see all the side effects of your code straight away, so why not?

File: flake.nix [hunk 3]
```diff
@@ -95,6 +95,12 @@
...
+            develop = true;
```
[the remark this answers is not in the document]
Comment (AndreyMlashkin) [reply flake.nix]: done

File: flake.nix (hunk 2 above)
[the remark this answers is not in the document]
Comment (akokoshn) [reply flake.nix:101]: build may takes more time, but ok for me

**Review by akokoshn — APPROVED** [review]

File: flake.nix (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndreyMlashkin) [reply flake.nix:101]: The idea of this derivation to be used by a developer for a local dev workflow.
CI should not use it

**Pull request merged**